src/console-logger.ts

```diff
@@ -0,0 +1,17 @@
+import { DefaultLogger } from '@tv2media/logger/node'
+import { Logger } from './logger'
+
+export class ConsoleLogger extends DefaultLogger implements Logger {
```

Comment:
The default implementation of the logger is a console logger. 
Would you rather have it named `LoggerImplementation`?

Reply:
No, I would prefer that it was made clear why this has gotten the name `ConsoleLogger`.
Right now it is because of something implicit in `@tv2media`. 
Optimally, `DefaultLogger` was called `ConsoleLogger` to imply it logs to the console. There is nothing stopping the authors of the library to change the implementation of `DefaultLogger` to something else than console and we would never know

Reply:
If we want it to be more explicit we can extend from the `NodeEnvironmentLogger` instead and set the console vault explicitly like so:

```typescript
export class ConsoleLogger extends NodeEnvironmentLogger implements Logger {
  private constructor() {
    super([
      new ConsoleVault({
        level: Level.TRACE,
        format: new PlainTextFormat(),
        isFormatLocked: false,
      })
    ])
  }
}
```

Reply:
Have applied Anders comment, such that we define the `ConsoleVault` ourselves, and extends from `NodeEnvironmentLogger` instead of `DefaultLogger`. This now makes it obvious, why it is called a `ConsoleLogger`.